src/components/proposal/ProposalTable.tsx

```diff
@@ -179,11 +182,13 @@ const ProposalTable = ({
           {
             icon: FileCopy,
             tooltip: 'Clone proposal',
-            onClick: (event, rowData) => {
-              setProposalToCloneId(
-                (rowData as PartialProposalsDataType).primaryKey
-              );
-              setOpenCallSelection(true);
+            onClick: (_event, rowData) => {
+              api()
```

Comment:
I think it is much better to extract this onClick logic/api call as a separate function outside of the `tsx` just for readability.

Reply:
The ProposalsTable should be refactored and all actions should be moved out. Did not want to make it part of this PR.